packages/flame/lib/src/effects/move_effect.dart

```diff
@@ -17,22 +19,26 @@ import 'transform2d_effect.dart';
 /// This effect applies incremental changes to the component's position, and
 /// requires that any other effect or update logic applied to the same component
 /// also used incremental updates.
-class MoveEffect extends Transform2DEffect {
+class MoveEffect extends Transform2DEffect implements MeasurableEffect {
   MoveEffect.by(Vector2 offset, EffectController controller)
-      : _offset = offset.clone(),
+      : vector = offset.clone(),
         super(controller);
 
   factory MoveEffect.to(Vector2 destination, EffectController controller) =>
       _MoveToEffect(destination, controller);
 
-  Vector2 _offset;
+  @protected
+  Vector2 vector;
```

Comment:
This could have a more descriptive name

Reply:
This is the best I could come up with. It's `vector` not because the type is `Vector2`, but because the amount of movement is literally described by a vector from the point of origin to the point of destination.

Do you have any better ideas for how to name it?

Reply:
Hmm, isn't the previous `offset` name better then?

Reply:
Looks like it doesn't even have to be protected, it's just a remnant from my older attempt at trying to build this feature another way. So i'll just revert it to how it was before.